Gordon360/ApiControllers/ScheduleControlController.cs

```diff
@@ -0,0 +1,196 @@
+﻿
+
+
+using System.Web.Http;
+using Gordon360.Models;
+using Gordon360.Services;
+using Gordon360.Repositories;
+using Gordon360.Models.ViewModels;
+using Gordon360.AuthorizationFilters;
+using Gordon360.Static.Names;
+using System;
+using Gordon360.Exceptions.ExceptionFilters;
+using Gordon360.Exceptions.CustomExceptions;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Linq;
+
+namespace Gordon360.Controllers.Api
+{
+    [RoutePrefix("api/schedulecontrol")]
+    [CustomExceptionFilter]
+    [Authorize]
+    public class ScheduleControlController : ApiController
+    {
+        private IUnitOfWork _unitOfWork;
+
+        //declare services we are going to use.
+        private IProfileService _profileService;
+        private IAccountService _accountService;
+        private IRoleCheckingService _roleCheckingService;
+
+        private IScheduleControlService _scheduleControlService;
+
+
+        public ScheduleControlController()
+        {
+            _unitOfWork = new UnitOfWork();
+            _scheduleControlService = new ScheduleControlService(_unitOfWork);
+            _accountService = new AccountService(_unitOfWork);
+        }
+
+        public ScheduleControlController(IScheduleControlService scheduleControlService)
+        {
+            _scheduleControlService = scheduleControlService;
+        }
+
+        /// <summary>
+        /// Get schedule information of local user
```

Comment:
Local user isn't very clear. Maybe logged in user?

Reply:
Change made